Rewrite the intro to IOTA. (#565)
iota-community/iota-wiki: internal/learn/about-iota/an-introduction-to-iota.md

---
id: an-introduction-to-iota
title: IOTA

> description: A basic introduction to the IOTA protocol.

Reviewer: ```suggestion
description: IOTA runs on the Tangle, a structure where newer transactions verify the older ones, and pretty much any other DLT runs on a blockchain instead.
```
We should also add an image tag to these files  so there's a nice thumbnail if they're shared :)   Off the top of my head a screenshot from the gifs could work, or a nice IOTA logo.

> description: A basic introduction to the IOTA protocol.

Reviewer: ```suggestion
description: IOTA runs on the Tangle, a structure where newer transactions verify the older ones
```

These articles could do with an `image` tag as well so there's a nice thumbnail if someone shares the link. 

Off the top of my head, it could be a screenshot from one of the gifs, or an IOTA logo.

---

# IOTA

> IOTA is a distributed ledger technology, a DLT. With the right approach, DLTs let us control our own [private data](https://wiki.iota.org/identity.rs/introduction), run [programs](https://wiki.iota.org/smart-contracts/overview) that nobody can meddle with, and trade and own assets without any middlemen or middlebanks.

Reviewer: ```suggestion
IOTA is a distributed ledger technology, a DLT. With the right approach, DLTs let us control our own [private data](https://wiki.iota.org/identity.rs/introduction), run [programs](https://wiki.iota.org/smart-contracts/overview) that nobody can meddle with, and trade and own assets without any intermediaries.
```

> IOTA is a distributed ledger technology, a DLT. With the right approach, DLTs let us control our own [private data](https://wiki.iota.org/identity.rs/introduction), run [programs](https://wiki.iota.org/smart-contracts/overview) that nobody can meddle with, and trade and own assets without any middlemen or middlebanks.

Reviewer: ```suggestion
IOTA is a distributed ledger technology, a DLT. With the right approach, DLTs let us control our own [private data](https://wiki.iota.org/identity.rs/introduction), run [programs](https://wiki.iota.org/smart-contracts/overview) that nobody can meddle with, and trade and own assets without any intermediaries.
```

> As in the name, a distributed ledger technology maintains a single ledger — a tally of token ownership — between multiple nodes. That would be trivial if all nodes were controlled by the same entity, but in DLTs the ledger state has to be agreed on by a collective of independent nodes. There is always a chance that a malicious node joins the network, so any DLT needs a way to protect itself. The way that IOTA takes makes it differ from other protocols.

Reviewer: ```suggestion
As in the name, a distributed ledger technology maintains a single ledger — a tally of token ownership — between multiple nodes. That would be trivial if the same entity controlled all nodes, but in DLTs the ledger state has to be agreed on by a collective of independent nodes. There is always a chance that a malicious node will join the network, so any DLT needs a way to protect itself. IOTA approaches this differently than protocols.
```

Reviewer 2: One suggestion from my side, remove the mention of " a DLT maintains a SINGLE ledger. Many DLTs handle multiple ledgers and sidechains / parachains. IOTA will do the same, so at one point it will become misleading. A DLT can manage multiple networks, of which each handles a single ledger of records of token ownership

Reviewer 2: And we have to have a strategy how to address the new nomenklatur. We decided to name everything blocks now.... How is this handled here? Messages should be completely removed from the wiki

> As in the name, a distributed ledger technology maintains a single ledger — a tally of token ownership — between multiple nodes. That would be trivial if all nodes were controlled by the same entity, but in DLTs the ledger state has to be agreed on by a collective of independent nodes. There is always a chance that a malicious node joins the network, so any DLT needs a way to protect itself. The way that IOTA takes makes it differ from other protocols.

Reviewer: ```suggestion
As the name implies, a distributed ledger technology maintains a single ledger — a tally of token ownership — between multiple nodes. That would be trivial if the same entity controlled all nodes, but in DLTs the ledger state has to be agreed on by a collective of independent nodes. There is always a chance that a malicious node will join the network, so any DLT needs a way to protect itself. IOTA approaches this differently than other protocols. 
```

> IOTA runs on [the Tangle](./tangle.md), a structure where newer transactions verify the older ones, and pretty much any other DLT runs on a blockchain instead. To secure its state and history, a blockchain has to collect transactions in blocks and chain the blocks one after another. That leads to a natural bottleneck: imagine loading the world's cargo into a single train wagon by wagon. IOTA bypasses it altogether.

Reviewer: ```suggestion
IOTA runs on [the Tangle](./tangle.md), a structure where newer transactions verify the older ones, and pretty much any other DLT runs on a blockchain instead. To secure its state and history, a blockchain must collect transactions in blocks and chain the blocks one after another. That leads to a natural bottleneck: imagine loading the world's cargo into a single train wagon by wagon. IOTA bypasses it altogether.
```

> The IOTA protocol is still in research. It has two public networks: the IOTA mainnet is the stable network that manages your IOTA tokens, and the Shimmer testnet is the staging network for the latest and greatest protocol updates. Once the changes are well tested on Shimmer, they come to the mainnet. The next big update is [Stardust](https://blog.shimmer.network/stardust-upgrade-in-a-nutshell/). The future update that concludes decentralization effort is called [Coordicide](roadmap-to-decentralization.md).

Reviewer: could you add links to IOTA mainnet and Shimmer ?

Author: I've added for the ../future/shimmer.md, but I'm not sure what to reference for the mainnet.

---

# IOTA

IOTA is a distributed ledger technology, a DLT. With the right approach, DLTs let us control our own [private data](https://wiki.iota.org/identity.rs/introduction), run [programs](https://wiki.iota.org/smart-contracts/overview) that nobody can meddle with, and trade and own assets without intermediaries.

As in the name, a distributed ledger technology maintains a single ledger — a tally of token ownership — between multiple nodes. That would be trivial if all nodes were controlled by the same entity, but in DLTs the ledger state has to be agreed on by a collective of independent nodes. There is always a chance that a malicious node joins the network, so any DLT needs a way to protect itself. The way that IOTA takes makes it differ from other protocols.

IOTA runs on [the Tangle](./tangle.md), a structure where newer transactions verify the older ones, and pretty much any other DLT runs on a blockchain instead. To secure its state and history, a blockchain must collect transactions in blocks and chain the blocks one after another. That leads to a natural bottleneck: imagine loading the world's cargo into a single train wagon by wagon. IOTA bypasses it altogether.

> The IOTA protocol is still in research. It has two public networks: the IOTA mainnet is the stable network that manages your IOTA tokens, and the [Shimmer testnet](../future/shimmer.md) is the staging network for the latest and greatest protocol updates. Once the changes are well tested on Shimmer, they come to the mainnet. The next big update is [Stardust](https://blog.shimmer.network/stardust-upgrade-in-a-nutshell/). The future update that concludes decentralization effort is called [Coordicide](roadmap-to-decentralization.md).

Reviewer 2: do not call it Shimmer testnet! It is not!
